build/crds/devices/devices_v1beta1_device.yaml

```diff
@@ -633,6 +633,40 @@ spec:
                                       type: object
                                   type: object
                               type: object
+                            openGemini:
```

Comment:
These extensions of `pushMethod` make the device instance CRD very large.

Can we abstract the configuration, for example, into a yaml string?
```yaml
pushMethod:
  type: influxdb2 / openGemini / http / mqtt / ...
  config: |
    hostName: ""
    port: 0
    requestPath: ""
    timeout: 0
```
Or reference a configuration CRD:
```yaml
# device CR
pushMethod:
  crd: http.devicepushmethod.kubeedge.io
  crname: xxx

# http device pushmethod CR
spec:
  hostName: ""
  port: 0
  requestPath: ""
  timeout: 0
```

Reply:
This is indeed a problem, and I will think about how to make it more lightweight